townhall/myapi/tests/test_volunteer.py

```diff
@@ -1,73 +1,108 @@
 from django.test import TestCase
+from django.core.management import call_command
+from unittest.mock import patch
 from myapi import models as townhall_models
-from myapi import services as townhall_services
 
 # VOLUNTEER
 
 # Testing ALL tests "python manage.py test myapi.tests"
 # Testing ONLY volunteer "python manage.py test myapi.tests.test_volunteer"
 
+@patch('myapi.services.VolunteerServices')
 class TestVolunteerModel(TestCase):
+    fixtures = ['volunteer_fixture.json']
+
     def setUp(self):
-        townhall_models.Volunteer.objects.create(id=1, first_name="Zamorak", last_name="Red", email="zamorak.red@gmail.com", gender="M")
-        townhall_models.Volunteer.objects.create(id=2, first_name="Guthix", last_name="Green", email="guthix_green@hotmail.ca", gender="F")
+        # Load the fixture data
+        call_command('loaddata', 'volunteer_fixture.json')
+
+    def test_get_volunteer(self, MockVolunteerServices):
+        # Mocking the get_volunteer method
+        MockVolunteerServices.get_volunteer.return_value = townhall_models.Volunteer(
+            id=1, first_name="Zamorak", last_name="Red", email="zamorak.red@gmail.com", gender="M"
+        )
```

Comment:
Read up a bit on some things that I think will be helpful here which are [unittest.mock](https://docs.python.org/3/library/unittest.mock.html) lib, specifically I would like to see `assert_called_once` `assert_not_called` and `assert_called_once_with` used where appropriate. These will level up these tests even further :D

Reply:
Sounds good to me! Thanks for the feedback and the suggestion. Excellent learning opportunity!